scripts/test_react_17

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh -e
+
+cp yarn.lock tmp.yarn.lock
+cp package.json tmp.package.json
+
+yarn add react@17.0.2 react-test-renderer@17.0.2 react-native@0.68.1 --ignore-scripts
```

Comment:
I hardcoded dependencies, but I assume we should be able to test other combination if we want, like different jest versions

Reply:
> Will this overwrite existing versions of react?

Yes.

> BTW these should be devDeps as currently we put them as such

Will fix that